class/defaults.yml

```diff
@@ -2,4 +2,29 @@ parameters:
   vshn_sli_reporting:
     =_metadata:
       multi_tenant: true
-    namespace: syn-vshn-sli-reporting
+    namespace: ${_instance}
+    namespaceMetadata: {}
+
+    images:
+      sli_reporting:
+        registry: ghcr.io
+        repository: vshn/vshn-sli-reporting
+        tag: v0.1.2
+
```

Comment:
One thing we usually have to touch first is setting resources correctly I'd add somthing like that:

```
resources:
  sli_reporting:
    requests: [...]
    limits: [...]
```